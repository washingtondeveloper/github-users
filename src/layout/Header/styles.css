.gs-header {
  height: 100px;
  padding: 15px;
  text-align: center;
}
