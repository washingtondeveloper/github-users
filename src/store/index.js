import { createStore, applyMiddleware } from "redux";
import reduxThunk from "redux-thunk";

import rootReducer from "./ducks";

export default createStore(rootReducer, applyMiddleware(reduxThunk));
