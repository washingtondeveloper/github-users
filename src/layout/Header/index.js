import React from "react";

import "./styles.css";
import Logo from "../../components/Logo";

export default () => (
  <header className="gs-header">
    <Logo />
  </header>
);
