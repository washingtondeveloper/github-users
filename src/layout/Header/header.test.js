import React from "react";

import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import store from "../../store";

import Header from "./";

describe("Component Header", () => {
  test("Deve renderizar sem erros", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Header />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
