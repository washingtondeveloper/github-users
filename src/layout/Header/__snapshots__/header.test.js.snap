// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Header Deve renderizar sem erros 1`] = `
<header
  className="gs-header"
>
  <div
    className="gs-logo"
  >
    <div>
      <img
        alt="github"
        src="git.png"
      />
    </div>
    <h1>
      GitUsers
    </h1>
  </div>
</header>
`;
